har/har_handlers.go

```diff
@@ -67,9 +68,15 @@ func (h *resetHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		log.Errorf("har: method not allowed: %s", req.Method)
 		return
 	}
-	h.logger.Reset()
 
-	log.Infof("resetHandler.ServeHTTP: HAR logs cleared")
+	if isBool, _ := strconv.ParseBool(req.URL.Query().Get("return")); isBool {
```

Comment:
handle the error

Reply:
What sort of error handling should be done here? I feel like with an optional query parameter it's typical to discard the value if it's invalid/unparsable

Reply:
I think sending a 400 back is what I'd expect. The client most likely meant to send something valid, and silently ignoring it would be confusing for debugging purposes. If you think that there's a reasonable case where the client would want an invalid value ignored, at least write out an log.Errorf for debugging.

Reply:
Makes sense, I added both a log line and 400 status